### DISCUSSION TITLE
Separating sub-sites of a multi-site installation to separate projects

### DISCUSSION THREAD

**Discussion by jasalt** [stayallive/wp-sentry#209] [Ideas]
Tracking errors from a multi-site works already, but with the default suggested configuration `define( 'WP_SENTRY_PHP_DSN', 'PHP_DSN' );` errors from all sub-sites are collected under one single project in GlitchTip (the same in Sentry I assume).

In issues list view the errors from different sub-sites cannot be separated, requiring eg. to look at the hostname header on each issues' detail page containing the sub-site domain which is cumbersome.

Simple workaround might be to create separate projects for individual sub-sites and then to customize setting the `WP_SENTRY_PHP_DSN` in `wp-config.php` dynamically on each request based on server name being requested to make different sub-sites use their own DSN (and maybe fallback to a multisite DSN if sub-site hasn't been separately added to Sentry/GlitchTip). I'll try that out for now, but other opinions would are also welcome.

**Comment by stayallive** [comment]
What about adding a tag which you can use to filter events for specific sites.

I would assume having all sites under a single installation in the same project to be helpful, especially so you don't get a issue occuring on every site many times but they are grouped correctly. With a tag you can still filter errors for a specific site but also get the benefit of issues being grouped.

However I'm not sure how GlitchTip handles this and if that is a feasable option because I have never used it.

**Reply by jasalt** [reply]
GlitchTip is API compatible but with some UI differences. With it, tag info cannot be shown on issue list page per entry, for search/filtering it should work..

I’ll learn more about those while testing, thank’s for tips.
